### PR TITLE
change new link to button

### PR DESCRIPTION
**Pull request by djamnazi**

### CODE REVIEW

File: BackofficeBundle/Resources/views/Tree/tree-macros.html.twig [hunk 1]
```diff
@@ -55,9 +55,10 @@
                     href="#"
                     data-url="{{ path('open_orchestra_backoffice_node_new', {'parentId': parentId}) }}"
                     data-type="node"
-                >
+                >   <span class="btn btn-success">
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/views/Tree/tree-macros.html.twig]: You should return before add span.

Comment (djamnazi) [inline BackofficeBundle/Resources/views/Tree/tree-macros.html.twig]: I really did that, clic on "view full changes" to see that.

Comment (itkg-nanne) [inline BackofficeBundle/Resources/views/Tree/tree-macros.html.twig]: In view mode, I've got ">   <span class="btn btn-success">" too. You should have :
>
<span ....

**Comment by djamnazi** [comment]
@itkg-nanne [OO-BUGFIX] PR updated

File: BackofficeBundle/Resources/views/Tree/tree-macros.html.twig [hunk 2]
```diff
@@ -96,8 +97,10 @@
                     data-url="{{ path('open_orchestra_backoffice_template_new') }}"
                     data-type="template"
                 >
+                    <span class="btn btn-success">
                     <i class="fa fa-file"></i>
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/views/Tree/tree-macros.html.twig]: You should indent after span.

File: BackofficeBundle/Resources/views/Tree/tree-macros.html.twig [hunk 3]
```diff
@@ -96,8 +97,10 @@
                     data-url="{{ path('open_orchestra_backoffice_template_new') }}"
                     data-type="template"
                 >
+                    <span class="btn btn-success">
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/views/Tree/tree-macros.html.twig]: I think that btn-success was not kept by Agnes.

**Comment by itkg-alavieille** [comment]
:+1:  but can you add the tag [OO-BUGFIX] by [OO-FEATURE] with a comment to explain the feature and add it in the first comment of your pull request.

**Comment by itkg-ngilain** [comment]
:+1: but as @itkg-alavieille your tag should be added at the top of this page

**Pull request merged**